Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.yml

```diff
@@ -86,6 +79,7 @@ script:
     - contextPath: ThreatGrid.Sample.CompletedAt
       description: The sample completion time
     description: Search samples on the Threat Grid platform. Input parameters are ANDed together. Only finished samples can be searched (that is, the ones that are having a status of succ or fail.)
+    deprecated: true
```

Comment:
please undo all the `deprecated: true` at the command level, leaving only the one at the yml root.